vsts/vsts.yaml

```diff
@@ -604,13 +597,10 @@ jobs:
           # TODO #181 Samples / tests fail the test due to fake connection strings.
           debugMode: false
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-vulnerabilityassessment.VulnerabilityAssessment@0
```

Comment:
##[warning]* This tool is no longer supported. Please remove this build task from your build definition. For more information, please see: https://aka.ms/gdn-v1-deprecation